README.md

```diff
@@ -22,6 +22,19 @@ cd jr
 make release.linux -j
 ```
 
+This is producing an executable named "jerry" in "build/bin/release.linux/".
```

Comment:
I'd merge this in the subsection below, e.g., "Once the executable `jerry` is built in `...` ...